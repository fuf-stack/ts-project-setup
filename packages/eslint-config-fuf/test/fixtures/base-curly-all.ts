/* eslint-disable */

if (foo) foo += 1;

while (bar)
    baz();

if (foo) {
    baz();
} else qux();
