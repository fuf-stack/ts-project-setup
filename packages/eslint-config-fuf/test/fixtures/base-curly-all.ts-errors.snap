[
  {
    "line": 1,
    "message": "Expected { after 'if' condition.",
    "ruleId": "curly",
  },
  {
    "line": 3,
    "message": "Delete `⏎···`",
    "ruleId": "prettier/prettier",
  },
  {
    "line": 4,
    "message": "Expected { after 'while' condition.",
    "ruleId": "curly",
  },
  {
    "line": 7,
    "message": "Delete `··`",
    "ruleId": "prettier/prettier",
  },
  {
    "line": 8,
    "message": "Expected { after 'else'.",
    "ruleId": "curly",
  },
]